evidence/pool.go

```diff
@@ -22,21 +22,23 @@ type Pool struct {
 	evidenceList *clist.CList // concurrent linked-list of evidence
 
 	// needed to load validators to verify evidence
-	stateDB dbm.DB
+	stateDB    dbm.DB
+	blockStore sm.BlockStore
```

Comment:
To validate DRB Evidence, which requires information the NextAeonStart and Qual stored in the block header